src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/git.rb

```diff
@@ -12,11 +12,13 @@ def self.last_tag
     tag
   end
 
```

Comment:
Non-blocking: let's add a doc string here to give the intention / context of this param.